backend/prompt_studio/prompt_studio_output_manager/output_manager_helper.py

```diff
@@ -139,7 +139,7 @@ def update_or_create_prompt_output(
                 challenge_data["file_name"] = metadata.get("file_name")
 
             output = outputs.get(prompt.prompt_key)
-            if prompt.enforce_type in {"json", "table"}:
+            if prompt.enforce_type in {"json", "table", "record"}:
```

Comment:
NIT: @jagadeeswaran-zipstack add a TODO to use enums here

Reply:
Sure